templates/e2e-llm-workflows/src/utils.py

```diff
@@ -2,10 +2,17 @@
 import os
 from tempfile import TemporaryDirectory
 import boto3
+from urllib.parse import urlparse
 from ray.data import Dataset
 
 
-def download_files_from_bucket(bucket, path, local_dir):
+def download_files_from_s3(s3_uri, local_dir):
```

Comment:
does this mean the e2e template can only be run on AWS clouds? should we call this out?

Reply:
Haha yes. The current code is meant only for AWS. 

It's an easy fix for me to make this work for GCP, don't know what the plan is though for ray summit.

Reply:
hmm yeah, I’d add GCP as well then, since ppl will still run this template even outside of Ray summit

Reply:
Added GCP! Will test now and update if needed!